### DISCUSSION TITLE
What are the supported games so far?

### DISCUSSION THREAD

**Discussion by CADIndie** [InoriRus/Kyty#19] [Q&A]
I am currently very interested in this project and I am willing to get not only myself but my team on it (as in contributing) but I first want to test it out and see what its limits and future are. Having a list of things like Games Support, Hardware Support, Bugs and as much info as you have would be amazing. Thank you in advanced and great work so far : )

**Comment by AboMedoz** [comment]
Worms WMD, Blackhole 
All the other games don't work so far

**Reply by CADIndie** [reply]
Do you have any more info? We need to know more of what's broken, partially integrated and any currently known bugs, as much info that you have pretty much. Without it we don't really have much reason to contribute to this over other projects.

**Reply by HalogenGam3r** [reply]
So far, it's trying to boot up Celeste, but it ended up with a unimplemented function.
